### PR TITLE
remove computed properties related stuff

### PR DESCRIPTION
**Pull request by sly7-7**
This should make all ember-try scenarios pass.
I had to update ember-cli to 3.28 for the beta and canary scenarios (avoid Ember is not defined) https://github.com/ember-cli/ember-cli/commit/cd0fa8b7b08887d60663fceb907623f37eb82619

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -235,73 +239,77 @@ Returns the time between `<dateA>` and `<dateB>` relative to `<dateB>` in a way
...
+| `<number>`  | Any value(s) [interpretable as a duration](https://momentjs.com/docs/#/durations/creating) by `moment` that is the amount of the `precision` you want to `add` to the `date` provided |
+| `precision` | An optional [unit of measurement](https://momentjs.com/docs/#/displaying/difference), defaults to `'milliseconds'`                                                                    |
 
 Mutates the original moment by adding time. See [`momentjs#add`](https://momentjs.com/docs/#/manipulating/add/).
 
 **Examples**
 
 ```hbs
-{{!-- Add 6 days to a date --}}
+{{! Add 6 days to a date }}
```
Comment (NullVoxPopuli) [inline README.md:277]: why these changes?

there are a lot of formatting changes in this PR that kinda distract from what's actually changing, ya know?

File: README.md [hunk 2]
```diff
@@ -552,7 +575,7 @@ export default Ember.Route.extend({
   moment: Ember.inject.service(),
   beforeModel() {
     this.get('moment').setLocale('es');
-  }
+  },
```
Comment (NullVoxPopuli) [inline README.md:578]: do you have an editor plugin or something that did this :thinking:

File: addon/helpers/moment-to-now.js [hunk 3]
```diff
@@ -1,23 +1,12 @@
-import { deprecate } from '@ember/application/deprecations';
-
 import computeFn from '../utils/helper-compute';
 import BaseHelper from './-base';
 
 export default BaseHelper.extend({
-  compute: computeFn(function (
```
Comment (NullVoxPopuli) [inline addon/helpers/moment-to-now.js:7]: nice! :raised_hands:

**Comment by NullVoxPopuli** [comment]
Noice! 
image

**Comment by sly7-7** [comment]
@NullVoxPopuli I don't know why, but I can't answer to your comments. Concerning the README, something happened during save. Because I've only removed parts concerning cps. I will revert the changes

**Comment by NullVoxPopuli** [comment]
thanks so much for doing this! <3
